addon/components/frost-button.js

```diff
@@ -69,11 +74,22 @@ export default Component.extend(PropTypeMixin, {
     text: PropTypes.string,
     title: PropTypes.string,
     type: PropTypes.string,
-    vertical: PropTypes.bool
+    vertical: PropTypes.bool,
+
+    // state
```

Comment:
Do we have this documented? It might be easier for people if it's documented. I think we should discuss this before removing those. Again I'm not speaking for everyone :) I'm speaking for me only. So It probably worth it to have a discussion.